TeamCode/src/main/java/org/firstinspires/ftc/teamcode/src/Vision.java

```diff
@@ -23,11 +26,18 @@ public void runOpMode() throws InterruptedException {
                 .setDrawTagOutline(true)
                 .build();
 
-        VisionPortal visionPortal = new VisionPortal.Builder()
-                .addProcessor(tagProcessor)
-                .setCamera(hardwareMap.get(WebcamName.class, "Webcam 1"))
-                .setCameraResolution(new Size(1280, 720))
-                        .build();
+        VisionPortal.Builder VisionPortalBuilder;
+        VisionPortalBuilder = new VisionPortal.Builder();
+
+        VisionPortalBuilder.addProcessor(tagProcessor);
+        VisionPortalBuilder.setCamera(hardwareMap.get(WebcamName.class, "Webcam 1"));
+        VisionPortalBuilder.setCameraResolution(new Size(1280, 720));
+        VisionPortalBuilder.setStreamFormat(VisionPortal.StreamFormat.YUY2);
```

Comment:
Based on the [FTC docs](https://ftc-docs.firstinspires.org/en/latest/apriltag/vision_portal/visionportal_init/visionportal-init.html), it looks like `StreamFormat.MJPEG` uses less bandwidth than `YUY2`. If network limitations were a problem for continuous streaming, this could be a good option.